openedx/core/djangolib/tests/test_js_utils.py

```diff
@@ -10,6 +10,9 @@
 
 from openedx.core.djangolib.js_utils import dump_js_escaped_json, js_escaped_string
 
+from django.utils.translation import ugettext_lazy as _
```

Comment:
`from django.utils.translation import ugettext_lazy as _` -> `from django.utils.translation import ugettext_lazy`, just to be explicit in tests.